### ISSUE TITLE
Can i choose another port? (not 8000)

### ISSUE THREAD

**Issue by ghost** [rndevfx/docker-blender-render-cluster#4]
My vps port 8000 was used by other application.

**Comment by cozybuild** [comment]
Yes you can (if you're running master/slave on the same machine). Simply use another port when you start the docker container. If you want to run it on port 9000, use:

`docker run --name blender_master -e "RENDER_MODE=MASTER" -p 9000:8000 -d d3v0x/blender-render-cluster`

Then you have to link the slave into the master. But if you're running the slave(s) on another host this shouldn't be possible right now. But I'll have a look at it and leave this issue open :)

**Comment by ghost** [comment]
yes, my issue is master and slave not running on same machine.
